src/tests/Microsoft.PowerFx.Interpreter.Tests/RecalcEngineAsyncTests.cs

```diff
@@ -19,14 +19,16 @@ namespace Microsoft.PowerFx.Tests
     // Test async eval features. 
     public class RecalcEngineAsyncTests : PowerFxTest
     {
-        // Trivial async function that runs synchronously. 
+        // Intentionally trivial async function that runs synchronously. 
+#pragma warning disable CS1998 // Async method lacks 'await' operators and will run synchronously
         private static async Task<FormulaValue> Worker(FormulaValue[] args, CancellationToken cancel)
         {
             var n = (NumberValue)args[0];
 
             var result = FormulaValue.New(n.Value * 2);
             return result;
```

Comment:
>result [](http://example.com/codeflow?start=19&length=6)

You can use `return Task.FromResult(result);` - and that would remove the need to use the `async` method modifier